Reconstruction/RecTracker/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ gaudi_add_test(CombinatorialTrackSeeding
 			         FRAMEWORK options/combinatorialTrackSeedingTest.py
 			         DEPENDS Examples.GeantFullSimTrackerHits)
 
-gaudi_add_test(ExtrapolationTest
-               WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
-			         FRAMEWORK options/extrapolationTest.py)
+# TODO: to be re-enabled once ACTS DD4hepPlugin is up to date
```

Comment:
maybe you can create an issue with this ToDo?

Reply:
#260